common/src/main/java/com/wynntils/models/gear/type/AbstractItemInfoDeserializer.java

```diff
@@ -0,0 +1,416 @@
+/*
+ * Copyright © Wynntils 2023.
+ * This file is released under LGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.models.gear.type;
```

Comment:
Not sure the type package is the right place for this. We usually reserve it for interfaces, enums and records.

Reply:
I think it better belongs in `models.wynnitem`, an "abstract" model I tried to create to unify similar item kinds. I think this has the same kind of vibe to it.